appsRendering.sh

```diff
@@ -0,0 +1,17 @@
+# scala package
+git clone https://github.com/guardian/content-api-models
+cd content-api-models
+cat ../appsRendering.thrift >> models/src/main/thrift/content/v1.thrift
```

Comment:
I think it might make sense to do that the other way around.

consider `content-api-models` as a folder that must reflect how the repo was when it was cloned.

Then you can read it, but shouldn't write anything in it. You can copy the files into a temporary directory if you want, then concatenate everything into one big thrift file

Also don't forget to add that folder to the .gitignore